### ISSUE TITLE
Unprotected resource

### ISSUE THREAD

**Issue by mavencode01** [sczyh30/vertx-blueprint-microservice#15]
Looks like the /api/* resources are not protected by keycloak 

```
 router.route("/api/*").handler(this::dispatchRequests);
```

**Comment by sczyh30** [comment]
Yes we don't protect all of the `/api/*` resources(e.g. retrieve actions). We only protect some actions requiring authentication. Here we protect resource by using a wrapped callback `requireLogin` in [RestAPIVerticle](https://github.com/sczyh30/vertx-blueprint-microservice/blob/master/microservice-blueprint-common/src/main/java/io/vertx/blueprint/microservice/common/RestAPIVerticle.java#L112). You may also refer to [authentication management section in the blueprint documentation](http://www.sczyh30.com/vertx-blueprint-microservice/api-gateway.html#implementing-api-gateway-with-vert-x).
